README.rst

```diff
@@ -11,3 +11,45 @@ images, similar to the pulp_docker plugin for Pulp 2.
 For more information, please see the `documentation
 <https://docs.pulpproject.org/pulp_container/>`_ or the `Pulp project page
 <https://pulpproject.org>`_.
+
+How to File an Issue
+--------------------
+
+`New pulp_container issue <https://pulp.plan.io/projects/pulp_container/issues/new>`_.
+
+.. warning::
+  Is this security related? If so, please follow the `Security Disclosures <https://docs.pulpproject.org/pulpcore/bugs-features.html#security-bugs>`_ procedure.
+
+Please set **only the fields in this table**. See `Redmine Fields <https://docs.pulpproject.org/pulpcore/bugs-features.html#redmine-fields>`_ for more detailed
+descriptions of all the fields and how they are used.
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+   :align: center
+
+   * - Field
+     - Instructions
+
+   * - Tracker
+     - For a bug, select ``Issue``, for a feature-request, choose ``Story``,
+       for a backport request, choose ``Backport``.
+
+   * - Subject
+     - Strive to be specific and concise.
+
+   * - Description
+     - This is the most important part! Please see `Description Field <https://docs.pulpproject.org/pulpcore/bugs-features.html#issue-description>`_.
+
+   * - Category
+     - Choose one if applicable, blank is OK.
+
+   * - Version
+     - The version of pulpcore that you discovered the issue.
```

Comment:
Shouldn't this be the pulp_container version?

Reply:
thank you, it was a bad copy/paste